Dockerfile

```diff
@@ -0,0 +1,4 @@
+FROM elixir:1.8.0-alpine
```

Comment:
Use the version specified here: https://github.com/ivx/ExKonsument/blob/master/.tool-versions#L1

Reply:
Thanks @mmainz - Fixed this. Copied from docker-compose where i changed it as well.